doc/discretization.rst

```diff
@@ -0,0 +1,282 @@
+unknowns, $\mathbf{Q}$ and $\mathbf{\Sigma}$:
+
+$$
+\mathbf{\Sigma} - \nabla{\mathbf{Q}} &= \mathbf{0}\quad \text{aux eqn}\\
+\frac{\partial \mathbf{Q}}{\partial t} + \underbrace{\nabla\cdot\mathbf{F}^I(\mathbf{Q}) -
+\nabla\cdot\mathbf{F}^V(\mathbf{Q},\mathbf{\Sigma})}_{= \nabla\cdot\mathbf{F}(\mathbf{Q},
+\mathbf{\Sigma})} &= \mathbf{S} \quad \text{primary eqn}
+$$
+
+Let $\Omega_h$ denote a collection of disjoint elements $E$. The DG method constructs
+approximations $\mathbf{Q}_h$ and $\mathbf{\Sigma}_h$ to $\mathbf{Q}$ and $\mathbf{\Sigma}$,
+respectively, in discontinuous finite element spaces. For any integer $k$, we define
+the following discontinuous spaces of piecewise (vector-valued) polynomial functions:
+$$
+\mathbf{V}^k_h &= \left\lbrace \mathbf{v} \in L^2(\Omega_h)^N \text{ such that }
+\mathbf{v}|_E \in \lbrack P^k(E) \rbrack^N, \text{ for all } E \in \Omega_h
+\right\rbrace, \\
+\mathbf{W}^k_h &= \left\lbrace \mathbf{w} \in L^2(\Omega_h)^{N\times d} \text{ such that }
+\mathbf{w}|_E \in \lbrack P^k(E) \rbrack^{N\times d}, \text{ for all } E \in \Omega_h
+\right\rbrace,
+$$
+where $N = d + 2 + N_s$, $d$ is the spatial dimension, and $N_s$ is the total number of
+mixture species. Here, $P^k(E)$ denotes a polynomial space on $E$ consisting of functions
+of degree $\leq k$. The DG formulation is obtained by multiplying by ''test functions''
+$\mathbf{v}_h \in \mathbf{V}^k_h$, $\mathbf{w}_h \in \mathbf{W}^k_h$
+(one for each equation repsectively) and integrating over each element.
+The resulting DG problem reads as follows. Find $(\mathbf{Q}_h,
+\mathbf{\Sigma}_h) \in \mathbf{V}^k_h \times \mathbf{W}^k_h$ such that, for all
+$(\mathbf{v}_h, \mathbf{w}_h) \in \mathbf{V}^k_h \times \mathbf{W}^k_h$, we have:
+
+$$
+\sum_{E\in\Omega_h} \left\lbrack \int_E \mathbf{v}_h\cdot\frac{\partial \mathbf{Q}_h}
+{\partial t} d\Omega + \oint_{\partial E} \mathbf{v}_h\mathbf{h} d\sigma - \int_E \nabla
+\mathbf{v}_h\cdot\mathbf{F}(\mathbf{Q}_h, \mathbf{\Sigma}_h)d\Omega\right\rbrack &=
+\sum_{E\in\Omega_h} \int_E \mathbf{v}_h\cdot\mathbf{S}_h d\Omega, \\
+\sum_{E\in\Omega_h}\left\lbrack \int_E \mathbf{w}_h\cdot \mathbf{\Sigma}_h d\Omega -
+\oint_{\partial E} \mathbf{w}_h\cdot\mathbf{H}_s d\sigma + \int_E \nabla\mathbf{w}_h\cdot
+\mathbf{Q}_h d\Omega\right\rbrack &= 0,
+$$
+
+where $\mathbf{h} = \mathbf{h}_e - \mathbf{h}_v$ is a *numerical flux* consisting of a
+*invicid* and *viscous* component respectively, and $\mathbf{H}_s$ is the
+*gradient numerical flux* for the auxiliary equation. Here, we use the subscript "h" to
+denote discretized quantities.
+
+Since $\mathbf{F}^I(\mathbf{Q}_h)\cdot\mathbf{n}$ is discontinuous, the quantities are
+allowed to vary on either side of a shared element boundary. That is, $\mathbf{F}^I
+(\mathbf{Q}^+_h)\cdot\mathbf{n}^+ \neq \mathbf{F}^I(\mathbf{Q}^-_h)\cdot\mathbf{n}^-$,
+where $\mathbf{n}^\pm$ denotes the ''cell-local'' (or interior) normal with respect to the
```

Comment:
```suggestion
where $\mathbf{n}^+$ denotes the exterior normal with respect to the
```
? (or explain both signs)
cc @thomasgibson

Reply:
I clarified in https://github.com/illinois-ceesd/mirgecom/pull/345/commits/8554dde3589abe8e3f97e437f0c178764e7b30a9